cardano-node/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -658,22 +658,18 @@ instance (Condense (HeaderHash blk), ProtocolLedgerView blk, SupportedBlock blk,
     TraceFoundIntersection _ _ _ ->
       mkObject [ "kind" .= String "ChainSyncClientEvent.TraceFoundIntersection" ]
 
-instance ToObject (TraceChainSyncServerEvent blk b) where
-    toObject _ _ev = mkObject [ "kind" .= String "ChainSyncServerEvent" ]
-  -- TODO: not yet exported from ouroboros-network
-  -- toObject verb ev = case ev of
-  --   TraceChainSyncServerIdle ->
-  --     mkObject [ "kind" .= String "ChainSyncServerEvent.TraceChainSyncServerIdle" ]
-  --   TraceChainSyncServerRead pt ->
-  --     mkObject [ "kind" .= String "ChainSyncServerEvent.TraceChainSyncServerRead"
-  --              , "block" .= toObject verb pt ]
-  --   TraceChainSyncServerReadBlocked pt ->
-  --     mkObject [ "kind" .= String "ChainSyncServerEvent.TraceChainSyncServerReadBlocked"
-  --              , "block" .= toObject verb pt ]
+instance StandardHash blk => ToObject (TraceChainSyncServerEvent blk b) where
+    toObject _ ev = case ev of
+      TraceChainSyncServerRead tip _ ->
+        mkObject [ "kind" .= String "ChainSyncServerEvent.TraceChainSyncServerRead"
+                 , "block" .= (String (pack . show $ tipPoint tip)) ]
+      TraceChainSyncServerReadBlocked tip _ ->
+        mkObject [ "kind" .= String "ChainSyncServerEvent.TraceChainSyncServerReadBlocked"
+                 , "block" .= (String (pack . show $ tipPoint tip)) ]
```

Comment:
This is incorrect (same on line 665), the `tip` reported by the `TraceChainSyncServerRead(Blocked)` is the current tip of the server's chain. The actually interesting bit is the `ChainUpdate`, which is either a new block that is added or a roll back to a point in the past.

Reply:
@mrBliss  I'll make the change in a follow up PR

Reply:
https://github.com/input-output-hk/cardano-node/pull/238